### DISCUSSION TITLE
Is it possible to start several wayvnc instances from one desktop session?

### DISCUSSION THREAD

**Discussion by mnlipp** [any1/wayvnc#182] [General]
I can pass different config files, but as wayvnc opens a socket, there might be a problem...

**Comment by lack** [comment]
If you mean the socket/TCP port it listens to for VNC connections, you can specify this on the command line or in the config file.

If you mean the new wayvncctl control socket, you can pass a new socket path on the command line via the `--socket` option which will have to be unique for each wayvnc process started by the same user.
